apps/webClient/src/components/NavButton/index.tsx

```diff
@@ -22,6 +24,7 @@ const useStyles = makeStyles((theme: Theme) => ({
   verticalBtn: {
     width: 60,
     height: 200,
+    marginTop: 40,
```

Comment:
I should have caught this earlier, no need to change but we should use `theme.spacing` here